layouts/partials/tw/image-rotator.html

```diff
@@ -5,7 +5,7 @@
 
 <span class="mx-auto block max-w-full" style="width: {{ .width }}px;">
   <span
-    class="{{ $bgClass }} relative block h-0 w-full overflow-hidden"
+    class="{{ $bgClass }} relative block h-0 w-full max-w-screen-xl mx-auto overflow-hidden"
```

Comment:
I haven't looked into this, so tell me if I'm wrong: shouldn't the max-w-screen-xl go on the container, because otherwise padding can be for wrong size? It might be time to rewrite this to use aspect-ratio to make the code more clear and cut out the middle span.

Reply:
This is where it needs to be. But yes, I can change to aspect ratio on another pr